Makefile.core.mk

```diff
@@ -78,6 +78,10 @@ docker: controller
 docker.push:
 	docker push $(HUB)/operator:$(TAG)
 
+docker.save:
+	mkdir -p /work/out/linux_amd64_release/docker
```

Comment:
this should probably be something like gopath/out?

Reply:
/work is where files are written within the container.  $GOPATH=/home/go in the docker container.  Not clear how to unify the ci system with the local build system...

Reply:
I don't think we should be hardcoding paths just because it works in the build container. istio/istio had some method to determine where out is, as does CNI, we should use the same method

Reply:
agreed re hardcoding.  Something is missing from our infrastructure.  CNI does not output to the local disk - istio is not on the containers plan.  It doesn't appear to output to the local disk either, even with its logic to detect the output location.

Reply:
OK, I screwed the output targets up. I started to put some stuff that would only work from the container. I'll fix these shortly. It'll require a new container and top-level makefile I think.

Reply:
istio/istio works like:
```
GO_TOP := $(shell echo ${GOPATH} | cut -d ':' -f1)
export OUT_DIR=$(GO_TOP)/out
export ISTIO_OUT:=$(OUT_DIR)/$(GOOS)_$(GOARCH)/$(BUILDTYPE_DIR)
```

Reply:
@howardjohn I tried that - it doesn't work in the operator implementation.  I doubt it works in istio/istio either...